report/report.go

```diff
@@ -40,14 +41,48 @@ func (h *HTMLReportUploader) DeployReports() []error {
 		h.logger.Printf("- %s", report.Name)
 	}
 
-	var errors []error
-	for _, report := range reports {
+	validatedReports, validationErrors := h.validate(reports)
+	if len(validationErrors) != 0 {
+		h.logger.Warnf("Validation errors:\n")
+
+		for _, validationError := range validationErrors {
+			h.logger.Warnf("- %s\n", validationError)
```

Comment:
I am not sure about the common practices but a header line might be useful?

Reply:
I did not add a header line because the validation happens after listing all of the found reports here:
https://github.com/bitrise-steplib/steps-deploy-to-bitrise-io/blob/2a49db699e82687b19591ebad80121cab539b03d/report/report.go#L32-L51

So my thinking was that we first list all of the reports and then we list the validation errors in the same header block because they belong together. I think helps them link together.